src/util/truffleConfig.js

```diff
@@ -16,6 +16,7 @@ function truffleConfig ({
   urlRinkeby = 'https://rinkeby.infura.io/', // 'http://node.rinkeby.gnosisdev.com:443',
   urlRopsten = 'https://ropsten.infura.io',
   urlMainnet = 'https://mainnet.infura.io',
+  urlXdai = 'https://xdai.poanetwork.dev',
```

Comment:
there is also https://dai.poa.network not sure if there is a difference 🤔

Reply:
https://www.xdaichain.com/for-developers/developer-resources#json-rpc-endpoints

The one I'm using is the main load balanced node.

Reply:
Seems the one I mentioned is the same just channeled through cloudflare (in the linked page it says `JSON RPC endpoint via Cloudflare` -> https://dai.poa.network)